x/xos/os_test.go

```diff
@@ -0,0 +1,91 @@
+package xos
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestTouch(t *testing.T) {
+	name := filepath.Join(os.TempDir(), "__test-file")
+	defer Remove(name)
+
+	if err := Touch(name); err != nil {
+		t.Fatalf("Touch failed: %s", err)
+	}
+	if !Exist(name) {
+		t.Fatalf("Touched file dosen't exist")
+	}
+
+	if err := Touch(name); err != nil {
```

Comment:
why we Touch twice?

Reply:
// Touch creates a new file, truncating it if it already exists. => to make sure touch doesn't' return an  error